woocommerce/woocommerce-gateway-stripe
# Enable UPE in `Add payment method` page

# Changes proposed in this Pull Request:

Issue: #1668

Description: Write a brief summary about this PR. As you compose your summary, consider each of these questions and address them if appropriate. Why is this change needed? What does this change do? Were there other solutions you considered? Why did you choose to pursue this solution? Describe any trade-offs you might have had to make.

<img width="782" alt="Screen Shot 2021-08-19 at 15 09 22" src="https://user-images.githubusercontent.com/407542/130121774-83d624cf-e1b8-4ff9-a189-b3cb4e67ba93.png">

# Testing instructions

- Enable UPE feature flag
- Enable UPE (Enable new checkout experience) in stripe settings
- Go to: `Shop` > `My Account` > `Payment methods`
- Click the `Add payment method` button
- Verify that the UPE form loads (as shown in the above screenshot)
- Fill the form with an invalid CC (ex: 4000000000000069)
- Check that an error message is displayed (and that the new payment method was not added)
- Fill the form with a valid CC (ex: 4242 4242 4242 4242)
- Check that the new payment method was added.
- Click the `Add payment method` button again
- Fill the form with a valid SCA CC (3D Secure)  (ex: 4000 0027 6000 3184)
- Check that the new payment method was added.
